asset/quickstart/batch-get-effective-iam-policies/main.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START asset_quickstart_batch_get_effective_iam_policies]
+
+// Sample asset-quickstart batch get effective iam policies.
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+
+	asset "cloud.google.com/go/asset/apiv1"
+	assetpb "google.golang.org/genproto/googleapis/cloud/asset/v1"
+)
+
+func main() {
+	scope := flag.String("scope", "", "Scope of the effective IAM policies to get.")
```

Comment:
Might be preferable to just pass in the projectID so that you can show in the sample how to construct the strings for scope and resource name.

https://googlecloudplatform.github.io/samples-style-guide/#minimal-arguments

Reply:
Good point. Changed.

Reply:
It looks like you could also construct fullResourceName using the projectID, unless you want to keep that general.

Reply:
I'd like to keep that general since fullResourceName can be different from Scope.